src/ode_solver/ode_solver_base.cpp

```diff
@@ -208,14 +208,13 @@ int ODESolverBase<dim,real,MeshType>::steady_state ()
     }
 
     if (ode_param.output_solution_vector_modulo > 0) {
-        for (unsigned int i = 0; i < this->dg->solution.size(); ++i) {
-            solutions_table.add_value(
-                    "Steady-state solution:",
-                    this->dg->solution[i]);
-        }
-        solutions_table.set_precision("Steady-state solution:", 16);
         std::ofstream out_file(ode_param.solutions_table_filename + ".txt");
-        solutions_table.write_text(out_file);
+        dealii::LinearAlgebra::ReadWriteVector<double> write_dg_solution(this->dg->solution.size());
```

Comment:
This is voluntary, I wanted to avoid having a header. I also currently only want the solution from the last run. `advance_solution_time()' is using a different and older code that outputs a header. I don't use this anymore so I could remove it if nobody else is using it?

Reply:
Good points, I have renamed them as you suggested. I have also removed the solution output from `advance_solution_time()` as no one seems to be using it.